api/src/services/webhooks.ts

```diff
@@ -1,32 +1,38 @@
+import { ErrorCode, createError } from '@directus/errors';
 import type { Bus } from '@directus/memory';
 import { useBus } from '../bus/index.js';
-import type { AbstractServiceOptions, Item, MutationOptions, PrimaryKey, Webhook } from '../types/index.js';
+import { useLogger } from '../logger.js';
+import type { AbstractServiceOptions, MutationOptions, PrimaryKey, Webhook } from '../types/index.js';
 import { ItemsService } from './items.js';
 
+const logger = useLogger();
```

Comment:
Can we move this into the constructor so it doesnt get initialized when the file gets imported, i know we wont be adding tests for this one as its deprecated but still good practice to prevent initializing things in the file root 😄

Reply:
I followed the convention of the other services which do it like this, then we might want to update all of them in a separate PR? :thinking:

Reply:
Hmm havent checked the other services in a bit, just looking from a testing point of view it's generally recommended to not initialize anything in the root context 🤔 But we dont have tests for the services do we 😬

Reply:
Is fine either way since the service will be completely dropped soon 👍